projects/packages/jetpack-mu-wpcom/src/features/wpcom-sidebar-notice/wpcom-sidebar-notice.php

```diff
@@ -124,6 +90,59 @@ function wpcom_get_sidebar_notice() {
 	);
 }
 
+/**
+ * Fetch sidebar notice data asynchronously via AJAX.
+ */
+function wpcom_fetch_sidebar_notice_data() {
+	check_ajax_referer( 'wpcom_fetch_sidebar_notice', 'nonce' );
```

Comment:
I think it should be:

```suggestion
	check_ajax_referer( 'fetch_sidebar_notice', 'nonce' );
```

Reply:
Fixed it in https://github.com/Automattic/jetpack/pull/40422/commits/a532e9499d4c8fca53e7b08afda654930d9beecd